### ISSUE TITLE
Extend Generic Work Type to Power IIIF Work Type

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#1854]
## Description
If it is possible to have a video dropped into a generic work type in Hyrax, a IIIF manifest will still need to be created for use with our player. Full descriptive metadata is out of scope for this issue, only critical metadata necessary to power a player.

In order to create an on the fly IIIF manifest, we will need to store critical technical and descriptive metadata. When this issue is complete Generic work will store all data needed to generate a IIIF manifest which our IIIF player can consume. 

### Notes 
- This can be done anytime after #2200 and isn't blocked by other iiif_manifest stories
- Not blocked by other metadata work
- Julie notes that we probably have all of this defined—we can check on this, but the fields are being used in Fedora 4 so we already have predicates for them.

### Related issues

Descriptive Metadata #1808

### Done Looks Like

- Generic Work can drive IIIF-AV in a player

## Tasks

- [x] Descriptive metadata necessary for manifest from #1808 is represented
- [x] Metadata SMEs give predicate for aspect ratio
- [ ] Display aspect ratio is now stored in the technical metadata for generic work
- [x] confirm duration is stored for all audio and video files
- [x] confirm height and width is stored for all video files
- [ ] confirm bitrate is stored for all audio and video files
- [ ] PRs all around. 

Note: You can't calculate display aspect ratio using height and width, you need to get it via FITS and store it.

**Comment by carrickr** [comment]
This is high priority for pointing at the next backlog buddies

**Comment by davidschober** [comment]
Currently it stores:
Notes: 

* Duration
* Height 
* Width
* We need to adjust for access ratio
* We  should store bitrate

We need to run a test with some other metadata types

This is an upstream hyrax change.

**Comment by jenyoung** [comment]
out of box descriptive metadata good to go for .1 work. .2 will have #1808 comparison.

**Comment by joncameron** [comment]
The main work for driving the IIIF player regarding technical metadata is adding support within Hyrax for those fields—captured in Hyrax. These are the fields that need to be extended on the Generic Work on the original file in the fileset.

**Comment by yingfeng-iu** [comment]
Looking at the fits 1.2.0 version of video xml (avi or mp4), bitRate and duration exist both in the video track component and its parent node (video) component, and their values are slightly different. In general, the video component contains both video track and audio tracks. I assume the values we want will be extracted from the video track component, not its parent node?

**Comment by joncameron** [comment]
"Display aspect ratio" vs "aspect ratio." These should be the same, in term of the metadata.
